src/main/java/de/caritas/cob/agencyservice/config/CacheManagerConfig.java

```diff
@@ -40,6 +42,30 @@ public class CacheManagerConfig {
   @Value("${cache.tenant.configuration.timeToLiveSeconds}")
   private long tenantTimeToLiveSeconds;
 
+  @Value("${cache.tenantId.configuration.maxEntriesLocalHeap}")
+  private long tenantIdMaxEntriesLocalHeap;
+
+  @Value("${cache.tenantId.configuration.eternal}")
+  private boolean tenantIdEternal;
+
+  @Value("${cache.tenantId.configuration.timeToIdleSeconds}")
```

Comment:
I think we don't need to have separate configs for each cache usecase. It can be one configuration for tenant cache and reused